tracing/README.md

```diff
@@ -5,7 +5,7 @@ An implementation exists for [Zipkin][]; [Appdash][] support is planned.
 
 [Dapper]: http://research.google.com/pubs/pub36356.html
 [Zipkin]: https://blog.twitter.com/2012/distributed-systems-tracing-with-zipkin
-[Appdash]: https://sourcegraph.com/blog/117580140734
+[Appdash]: https://sourcegraph.com/github.com/sourcegraph/appdash
```

Comment:
I didn't know any about `appdash` and I was in airport shuttle with a crappy WIFI connection, so I made a duckduckgo search and I got that link.

But, you're right, better to point to github :smile: